components/StudentCard/StudentCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Button, Text, TouchableHighlight, View } from 'react-native';
+import { APIRoutes } from '../../config/routes';
+import PropTypes from 'prop-types';
+
+/**
+ * @prop student - student information
+ * @prop onSelectStudent - callback function to show student information
+ */
+class StudentCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Don't really need to set state like this, since you just use this.props.student anyways